dangerzone/conversion/pixels_to_pdf.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+"""
+Here are the steps, with progress bar percentages:
+
+- 50%-95%: Convert each page of pixels into a PDF (each page takes 45/n%, where n is the number of pages)
+- 95%-100%: Compress the final PDF
+"""
+import asyncio
+import glob
+import json
+import os
+import shutil
+import sys
+
+# try:
+#     from .common import DangerzoneConverter, run_command
+# except:
+#     # Importing without a module is needed when this runs in a container
+#     from common import DangerzoneConverter, run_command, running_on_qubes
```

Comment:
Shall we remove this?

Reply:
Actually, we can't https://app.circleci.com/pipelines/github/freedomofpress/dangerzone/1392/workflows/cceaf5a5-c015-4b28-89fb-fdf24a4fb5f9/jobs/12814

Reply:
Well, this try-import clause is bad, because it introduces ambiguity. But you're right that removing it breaks the container isolation provider. I think fixing the entry point, so that we can do `python -m` is much more cleaner. I'll take a look at it.

Reply:
I reverted it to the try...import clause so it would run (https://github.com/freedomofpress/dangerzone/commit/e0e5c923a3ba686214a169cb9c2778b7546dc4fa). Leaving this open in case you want to implement another solution.

Reply:
We now have a fix for that in 8841d534. We install Dangerzone in the container and disposable VM in such a way that we can import it as a package, meaning that relative imports work now everywhere.